docs/oauth.md

```diff
@@ -1,98 +1,28 @@
 ---
-title: OAuth Authentication Support
+title: OAuth Configuration and Usage
 ---
 **IMPORTANT:** OAuth for plugins is available in Tableau 2021.1 and newer. Multi-IDP config for plugins is available starting in Tableau 2023.1.
 
+This document explains how to configure and use OAuth. It's meant for admins and plugin developers. For information on adding OAuth to a connector plugin, see [OAuth Development](./oauth-dev.md).
```

Comment:
Question: is the SDK documentation the right place for “general” OAuth custom config documentation? Or should that go to the main Tableau documentation? The SDK docs have generally been targeted solely for people developing connector plugins, not for the general Tableau audience. 

You say the following information is for "admins and plugin developers", yet separate out the "OAuth configuration and Usage" with "OAuth for plugins". I argue that the SDK documentation should only have information related to building and supporting connector plugins, anything else should go to the Tableau documentation.

Instead of adding this to the SDK doucmentaiton, maybe we should get it added to the generic OAuth documentation: https://help.tableau.com/current/server/en-us/protected_auth.htm

Reply:
I will submit this and open a follow up story to have the content copied into the main docs. After that is complete we can redirect.